.circleci/config.yml

```diff
@@ -1,5 +1,42 @@
 version: 2.1
 jobs:
+    lint:
+        docker:
+            - image: cimg/base:stable
+              auth:
+                  username: mydockerhub-user
+                  password: $DOCKERHUB_PASSWORD
+              environment:
+                  CMAKE_BUILD_TYPE: Release
+                  FLAGS: -Wall -Wextra -Werror
+                  OSMT_INSTALL: ~/osmt-install
+                  USE_READLINE: OFF
+
+        steps:
+            - checkout
+            - run:
+                  name: apt-get update
+                  command: sudo apt-get update
+            - run:
+                  name: install dependencies
+                  command: sudo apt-get install libgmp-dev libedit-dev bison flex cmake libubsan0 python3-pip cython
+            - run:
+                  name: install model validation environment
+                  command: ./regression_models/bin/setup-env.sh
+            - run:
+                  name: Install clang
+                  command: |
+                      sudo apt-get install clang
+                      sudo update-alternatives --install /usr/bin/c++ c++ /usr/bin/clang++ 60
```

Comment:
I think all these steps are not necessary and can be removed, no?

Reply:
Actually I think that's right, not sure about clang, but we may check